src/mozilla_sec_eia/models/sec10k/__init__.py

```diff
@@ -37,39 +48,76 @@
 ex21_production_job = model_jobs.create_production_model_job(
     "ex21_extraction",
     ex_21.production_assets,
-    concurrency_limit=4,
+    tag_concurrency_limits=[
+        {"key": "model", "value": "exhibit21_extractor", "limit": 2},
+    ],
 )
 
-ex21_validation_job = model_jobs.create_validation_model_job(
-    "ex21_extraction_validation",
-    ex_21.validation_assets,
+
```

Comment:
At some point I'd like to clean up all this asset/job creation stuff, but I don't think this is the best time for that.